src/OI.cpp

```diff
@@ -43,7 +47,11 @@ OI::OI()
 
   SmartDashboard::PutData("TankDrive", new TankDrive());
   SmartDashboard::PutData("Autonomous Command", new AutonomousCommand());
-
+  SmartDashboard::PutData("FeederOn", new ConveyorControl(1.0, 0.0));
```

Comment:
The string "FeederOn" in this case is used in the SmartDashboard user interface as a title for the control. I think you can actually have spaces and use real English and capitalization and everything. 😈 

Perhaps "Feeder On", "Mover On"? or is that just weird?